rasa/core/channels/audiocodes.py

```diff
@@ -0,0 +1,250 @@
+import logging
+from rasa.utils.common import raise_warning
+from sanic import Blueprint, response
+from sanic.request import Request
+import sanic
+from typing import Text, List, Dict, Any, Callable, Awaitable, Iterable, Optional
+from rasa.core.channels.channel import UserMessage, OutputChannel, InputChannel
+from sanic.response import HTTPResponse
+import json
+import uuid
+import datetime
+from rasa.core import jobs
+from rasa.core.constants import INTENT_MESSAGE_PREFIX
+
+logger = logging.getLogger(__name__)
+
+CHANNEL_NAME = "audiocodes"
+KEEP_ALIVE_SECONDS = 120
```

Comment:
Is there a way to add channel configuration instead of this constant?

Reply:
you can do that in `from_credentials` as done here https://github.com/RasaHQ/rasa/blob/fbac3387d661614439aab37d4b5142b266b8b8a4/rasa/core/channels/telegram.py#L152 or e.g. use an env variable